modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -480,6 +625,152 @@ public int getZoneId(String name) {
         }
     }
 
+    /**
+     * The method for obtaining data nodes of the specified zone. If the dataNodesAutoAdjustScaleUp or dataNodesAutoAdjustScaleDown
```

Comment:
We need links to `dataNodesAutoAdjustScaleUp`, `dataNodesAutoAdjustScaleDown`. User must be able to understand what these concepts mean.

Reply:
Fixed.